README.md

```diff
@@ -58,6 +92,25 @@ You can also run the samples in Docker (see below).
 
 ### Configuring the sample to use SQL Server
 
+1. In the `src/Web/Program.cs` file, switch to the `Locally` configuration of SQL Server.
```

Comment:
@zedy-wj - Can you please update the app so that no code changes are required to run this locally?

Reply:
@jongio - We added a parameter in [appSettings](https://github.com/dotnet-architecture/eShopOnWeb/pull/844/files#diff-7ef659fc9cf6968e718894d300490b14ea7a52091e7d4bcffae3a5029ac721d4R63) to distinguish different environments of `locally` and `Azd template`.